src/.vuepress/theme/client/components/global/OpenApiSchemaInfo.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <header>
+    <strong v-if="title">{{ title }}</strong>
+    <strong v-else-if="schema && schema.title">{{ schema.title }}</strong>
+    <span
+      v-if="isRequired"
+      title="Required"
+      class="text-red-600"
+      >*</span
+    >
+    <pre
+      v-if="schema && schema.type"
+      class="m-0 p-0 text-sm whitespace-nowrap">
+          {{ schema.type }}
+          <template v-if="schema.format">({{ schema.format }})</template>
+      </pre>
+
+    <Markdown
+      v-if="schema && schema.description"
+      :content="schema.description"
+      class="m-0 p-0 text-gray-500 text-sm" />
+
+    <div v-if="schema && schema.enum?.length">
+      <strong class="text-sm">Possible values:</strong>
+      <ul
+        v-if="schema.enum.length < 10"
+        class="columns-2 list-disc">
+        <li
+          v-for="value in schema.enum"
+          :key="value">
+          <code>{{ value }}</code>
+        </li>
+      </ul>
+      <details v-else>
+        <summary>View all</summary>
+        <ul class="columns-2 list-disc">
+          <li
+            v-for="value in schema.enum"
+            :key="value">
+            <code>{{ value }}</code>
+          </li>
+        </ul>
+      </details>
+    </div>
+
+    <div v-if="schema.externalDocs">
```

Comment:
moeten we hier dan ook niet checken of `schema` bestaat? want dat doen we hierboven wel.